### ISSUE TITLE
Test: Add function to calculate power loss

### ISSUE THREAD

**Issue by tangjk999** [tangjk999/buck-boost-calc-tools#1]
Please add a function to calculate power loss in the Buck-Boost converter based on switching frequency and MOSFET resistance.

**Comment by tangjk999** [comment]
approved - please implement this feature

**Comment by tangjk999** [comment]
🤖 I had trouble generating code for this issue. Please provide more details or handle it manually.

**Comment by tangjk999** [comment]
## 已实现功率损耗计算功能

已添加 `calculatePowerLoss` 函数，基于开关频率和MOSFET导通电阻计算功率损耗。

**函数签名:**
```javascript
calculatePowerLoss(fSw, rDSon, iSW, vSW, duty)
```

**参数:**
- `fSw` - 开关频率 (Hz)
- `rDSon` - MOSFET导通电阻 (mΩ)
- `iSW` - 开关电流 (A)
- `vSW` - 开关电压 (V)
- `duty` - 占空比 (0-1)

**返回值包含:**
- `pConduction` - 导通损耗 (W)
- `pSwitching` - 开关损耗 (W)
- `pGate` - 栅极驱动损耗 (W)
- `pTotal` - 总功率损耗 (W)
- `summary` - 损耗摘要字符串

**使用示例:**
```javascript
const loss = BuckBoostCalculator.calculatePowerLoss(100000, 2.5, 20, 400, 0.12);
// 返回: { pConduction: 0.12W, pSwitching: 0.4W, pGate: 0.1W, pTotal: 0.62W, ... }
```

提交: `d8fe011`

**Comment by tangjk999** [comment]
这是一个测试用的issue，可以不做任何修改
